www/annotationdb/annotationdb.css

```diff
@@ -0,0 +1,443 @@
+div.wy-nav-content{
```

Comment:
I've actually left the CSS that increases the max-width of the div from 800 to 1000px. I think it helps the layout of the query builder tree. The left margin should be in line with the other doc pages, though.